src/Language/PureScript/Errors.hs

```diff
@@ -1333,7 +1333,7 @@ renderBox = unlines
 
 toTypelevelString :: Type -> Maybe Box.Box
 toTypelevelString (TypeLevelString s) =
-  Just . Box.text $ decodeStringWithReplacement s
+  Just . line . T.strip $ decodeStringWithReplacement s
```

Comment:
This isn't going to handle errors which span multiple lines very well. If we're going to strip spaces at all, we should strip the same number from each line.